NCsvPerf.Test/CsvReadable/Benchmarks/PackageAssetsSuiteTest.cs

```diff
@@ -41,15 +41,15 @@ public void AllBenchmarksHaveSameOutput(int lineCount)
 
             // Assert
             var groups = results
-                .GroupBy(p => JsonConvert.SerializeObject(p.Value, Formatting.Indented), p => p.Key)
+                .GroupBy(p => JsonConvert.SerializeObject(p.Value, Formatting.Indented).Length, p => p.Key)
```

Comment:
In general, I think the test should be asserting the full parsed output, rather than just the length.

Could we add a "sort" step to the end of your `RecordParser.cs` based on a collected line number/byte offset?

Most CSV parsers I have seen yield results in the same order as the file some source data may have meaning baked into the ordering (e.g. a ranked list).

Reply:
> Could we add a "sort" step to the end of your RecordParser.cs based on a collected line number/byte offset?

I tried to add a sort before serialize the list here, but unfortunately I could not find a combination of fields that ensure all list become equals. Theres a lot of duplicated values in the csv.

> Looks like the unit test is failing. Could you take a look?

I adjusted the code to respect lines order. But just for curiosity, is order an necessary or optional part for the benchmark?

> Latest results (before merge) are available here: https://www.joelverhagen.com/blog/2020/12/fastest-net-csv-parsers

I meant if you could run the benchmark for this branch before merge, if possible. Just to check if your results are to much different of mine.